forge-ai/src/main/java/forge/ai/PlayerControllerAi.java

```diff
@@ -762,7 +762,7 @@ public void notifyOfValue(SpellAbility saSource, GameObject realtedTarget, Strin
     @Override
     public boolean chooseBinary(SpellAbility sa, String question, BinaryChoiceType kindOfChoice, Boolean defaultVal) {
         switch (kindOfChoice) {
-            case TapOrUntap: return true;
+            case TapOrUntap:
             case UntapOrLeaveTapped:
```

Comment:
i don't think they can be merged for AI this way 

the return value is different for both of them

Reply:
This was just a bad rollback, will fix